ticker/src/main/java/com/robinhood/ticker/TickerView.java

```diff
@@ -229,9 +233,11 @@ public int getTextColor() {
      * @param color the color to set the text to.
      */
     public void setTextColor(int color) {
-        textColor = color;
-        textPaint.setColor(textColor);
-        invalidate();
+        if (this.textColor != color) {
+            textColor = color;
+            textPaint.setColor(textColor);
+            invalidate();
```

Comment:
I realized that `onTextPaintChanged` was a poorly named method. It should really be `onTextPaintMeasurementChanged`. I updated the method's name and left this as `invalidate()`.